src/components/labels.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+
+interface LabelProps {
+  hideText: string;
+  modalLabelClass: string;
+  modeText?: string;
+  liftState?: Readonly<RomiCore.LiftState>;
+  liftName?: string;
+  name?: string;
+}
+
+export default function Labels(props: LabelProps) {
```

Comment:
`Labels` feel like a too generic name. Can we be more descriptive with that name it something like OmnipanelStateLabel?